zerver/management/commands/convert_mattermost_data.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+import os
+from typing import Any
+
+'''
+Example usage for testing purposes:
+
+    ./manage.py convert_mattermost_data mmbulk_data_mar8_2019.json --output mm_export
+    ./manage.py import --destroy-rebuild-database mattermost mm_export
+
+Test out the realm:
+    ./tools/run-dev.py
+    go to browser and use your dev url
+
+spec:
+https://docs.mattermost.com/administration/bulk-export.html
```

Comment:
Let's move that spec link to a comment at the top of zerver/data_import/mattermost.py ; I think that's more helpful.

Can you also add a file similar to templates/zerver/help/import-from-slack with both docs for using this and an accurate caveats list (e.g. we should be clear that Mattermost doesn't support exporting uploaded files, and that's why they aren't included).  We will eventually want to purge everything from the caveats list that's our fault.